### PR TITLE
add receiver method ToURL to *Options

### PR DESCRIPTION
**Pull request by dillonstreator**
discussion reference https://github.com/go-pg/pg/discussions/1933

### CODE REVIEW

File: options.go [hunk 1]
```diff
@@ -256,6 +256,53 @@ func ParseURL(sURL string) (*Options, error) {
...
+		values.Add("application_name", opts.ApplicationName)
+	}
+
+	if opts.TLSConfig == nil {
+		values.Add("sslmode", "disable")
+	} else if opts.TLSConfig.InsecureSkipVerify {
+		values.Add("sslmode", "allow")
+	} else if !opts.TLSConfig.InsecureSkipVerify {
+		values.Add("sslmode", "verify-ca")
+	}
```
Comment (elliotcourant) [inline options.go:296]: An else at the end that also includes `sslmode=disable` might not be a bad idea?

File: options_test.go [hunk 2]
```diff
@@ -261,3 +261,29 @@ func TestParseURL(t *testing.T) {
 		})
 	}
 }
+
+var hello struct {
+	Test string
+}
+
+func TestOptions_ToURL(t *testing.T) {
```
Comment (elliotcourant) [inline options_test.go:267]: Another good test case might be to try to parse the URL that is output from ToURL to make sure that it is still a valid input.

**Review by elliotcourant — CHANGES_REQUESTED** [review]

File: options_test.go (hunk 2 above)
Comment (dillonstreator) [reply options_test.go:267]: yeah that makes sense to me.. good call

File: options.go (hunk 1 above)
Comment (dillonstreator) [reply options.go:296]: @elliotcourant an `else` branch wouldn't be possible to hit seeing that if `TLSConfig` is not `null`, given the zero value of `bool`, either of the `else if` cases would be guaranteed to execute.

**Review by elliotcourant — APPROVED** [review]

**Pull request merged**